units/undead/Avatar_of_Ending.cfg

```diff
@@ -0,0 +1,297 @@
+#textdomain wesnoth-Flight_Freedom
+
+#sprites adapted from Dark Master project
+[unit_type]
+    id=Avatar of Ending
+    name= _ "Avatar of Ending"
+    race=undead
+    ignore_race_traits=yes
+    {TRAIT_UNDEAD}
+    {TRAIT_QUICK}
+    {TRAIT_RESILIENT}
+    {TRAIT_STRONG}
+    num_traits=2
+
+    image="units/undead/avatar-of-ending.png"
+    profile="portraits/avatar-of-ending.png"
+    small_profile="portraits/avatar-of-ending.png~SCALE(205,205)"
+    hitpoints=138
+    movement_type=undeadspirit
+    movement=8
+    experience=140
+    level=6
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=190
+    usage=warrior
+    description= _ "Formerly one of the greatest of drakes, this Avatar of Ending is but a contradiction. What were once radiant internal fires are now empty flames of oblivion, impossibly animating an entity singularly dedicated to non-existence. Its ability to End its foes is little short of absolute.
+
+    A new age beckons, the End of Everything; though whether this is truly a new age or the ending of the ages is a question beyond mortal minds. This Avatar will surely bring about the End unless stopped. But one must ask - how can an Ending truly be brought to its end?"
+    [levelout_anim]
+        [frame]
+            duration=600
+            blend_ratio="0~1:600"
+            blend_color=0,0,0
+        [/frame]
+    [/levelout_anim]
+    [levelin_anim]
+        [frame]
+            duration=600
+            blend_ratio="1~0:600"
+            blend_color=0,0,0
+        [/frame]
+    [/levelin_anim]
+    [death]
+        start_time=0
+        [portal_frame]
+            duration=600
+            image="halo/particle-anims/particle-c-[01~36].png"
+            image_mod="~SCALE(150,184)~BLEND(255,0,0,1)"
+            alpha=1.0:300,1.0~0.0:300
+            auto_vflip=false
+            offset=0.0
+            y=-12
+            layer=99
+            sound=chaotic-blast.wav
+        [/portal_frame]
+        #        image_mod="~CS():300,~BLEND(0,0,0,0.75)~O(50%)"
```

Comment:
why's this line commented out?

Reply:
I think it was commented out in the original code that this was based on (from Dark Master project).

Reply:
Personally I prefer that code either be ifdef-ed out (so that it can be re-enabled more easily), or just removed entirely, instead of commented out... which of the 2 do you think would make more sense here?

Reply:
probably just remove it entirely for now

Reply:
```suggestion
```

Reply:
ah ok I see that was included in [ac8aee5](https://github.com/cooljeanius/Flight_Freedom/pull/18/commits/ac8aee5e6fb3d1bf231b3f5053e383c7ef0e26ef) now...